docs/guide/execution/quickrun_execution.rst

```diff
@@ -44,6 +44,69 @@ The ``quickrun`` command can be integrated into as:
 
   openfe quickrun transformation.json -o results.json
 
+Parallel execution of repeats with Quickrun
+===========================================
+
+Serial execution of multiple repeats of a transformation can be inefficient when working with a HPC, in this case higher
+throughput can be achieved by running one repeat per HPC job allowing for parallel execution. Most protocols are setup to
+run ``three repeats in seral`` by default, however, this can be changed either via the protocol setting ``protocol_repeats``, see the
+:ref:`protocol configuration guide <cookbook/choose_protocol.nblink>` for more details or overridden via the
+``--n-protocol-repeats`` flag of the ``openfe quickrun`` command. Each transformation can then be executed multiple times via the
```

Comment:
```suggestion
run three repeats in serial by default, but this can be changed by either:
 
 1. Defining the protocol setting ``protocol_repeats`` - see the :ref:`protocol configuration guide <cookbook/choose_protocol.nblink>` for more details.
 2. Using the ``openfe plan-rhfe-network`` (or ``plan-rbfe-network``) command line flag ``--n-protocol-repeats``.  Each transformation can then be executed multiple times via the
```

Reply:
I think breaking this out into a list might be easier for users to find visually - also this is a planning-stage flag, not a quickrun flag.

Reply:
yeah that is much easier and wow good catch!